server_test.go

```diff
@@ -24,6 +24,8 @@ var (
 	address = lnwire.NetAddress{IdentityKey: pubKey, Address: &StubAddress{}}
 )
 
+// Non-permanent connects should result in the peer being added to
```

Comment:
The comments added in this commit should be prefixed by the _name_ of the test being commented. Thanks!